configuration/db_mongo.rst

```diff
@@ -64,7 +64,7 @@ This works the same as with the normal Firely Server database, except that you:
      "Administration": {
          "Repository": "MongoDB",
          "MongoDbOptions": {
-             "ConnectionString": "mongodb://localhost/vonkstu3",
+             "ConnectionString": "mongodb://localhost/vonkadmin",
```

Comment:
When you are already changing the databasename, then this is the time to get rid of the name `vonk`, right?